holopy/scattering/tests/test_composite.py

```diff
@@ -40,8 +40,12 @@ def test_all_ties(self):
 
     @attr("fast")
     def test_raw_parameters(self):
-        spheres = [Sphere(n=np.random.rand(), r=np.random.rand(),
-                          center=[i, i, i]) for i in range(3)]
+        max_radius = np.sqrt(3) / 2.0
```

Comment:
@barkls just a side comment -- we were talking about whether or not to set a seed during a test; this one is a good example. Pre-3f5a79b, the max radius was (implicitly) 1 which is > sqrt(3) / 2 and could lead to some overlaps. However, it just happened that the radii that were drawn were less than sqrt(3) /2, so the old version of this test would pass. It would pass every time because the tests are run in the same order and the random seed is set by some of the tests. However, when I factored out a different test, it changed the order the tests were run in, which changed the random values of the radii and happened to lead to a failure. Cool story, huh?

Reply:
Neat find! Thanks for fixing.